hieradata_aws/common.yaml

```diff
@@ -610,6 +610,7 @@ govuk::apps::govuk_crawler_worker::root_urls:
   - "https://assets.%{hiera('app_domain')}"
   - "https://www.%{hiera('app_domain')}"
 
+govuk::apps::imminence::mongodb_name: 'imminence_production'
```

Comment:
why not keep it as the default db name in `[modules/govuk/manifests/apps/imminence.pp](https://github.com/alphagov/govuk-puppet/pull/11573/files#diff-5b5aa948d59a4b8ab656f91eee5d216d21aade137e36c531106d581d32b23847)`

Reply:
I get it, I would have kept it in the class definition, to be honest. They are the same across all environments. Up to you.

Reply:
If it's the same everywhere, class definition makes more sense. I'll remove it from here and from the /integration/imminence.yaml file.